monitoring/parse_monitoring_log.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+
+def parse_task_log(file_iterator):
+    """ identify allocated and used memory and disk space within a cromwell task
+
+    :param iterator file_iterator: output generated by monitoring.sh script run via
+      cromwell on a google compute VM. Most often called 'monitoring.log', and found in
+      a task's cromwell directory within cromwell executions.
+    :return int max_memory: maximum memory usage within task
+    :return int total_memory: memory allocated for task
+    :return int max_disk: maximum disk usage within task
+    :return int total_disk: disk space allocated for task
+    :return bool robust_estimate: True if there were 5 or more estimates across the
+      task's run.
+
+    """
+    total_memory = None
```

Comment:
Just wondering, is there a reason `total_memory` and `total_disk` are `None` and not `0`?